server/src/main/java/com/decathlon/ara/cartography/SquashExporter.java

```diff
@@ -1,16 +1,43 @@
 package com.decathlon.ara.cartography;
 
+import com.decathlon.ara.domain.enumeration.FunctionalityType;
 import com.decathlon.ara.service.dto.functionality.FunctionalityDTO;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.poi.hssf.usermodel.HSSFWorkbook;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.CellStyle;
+import org.apache.poi.ss.usermodel.CreationHelper;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
 
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 
 /**
  * SquashExporter is an Exporter which serialize the functionalities in order to make them importable in SquashTM.
  *
  * @author Sylvain Nieuwlandt
  * @since 4.1.0
  */
+@Slf4j
 public class SquashExporter extends Exporter {
+
+    private static final String PROJECT_NAME = "squash_project_name";
+    private static final String USER = "squash_user";
+    private static final Map<String,String> CRITICALITY_MAPPING = new HashMap<>();
+
+    static {
+        CRITICALITY_MAPPING.put("HIGH", "CRITICAL");
```

Comment:
What do you think about using an Enum ? Final variables ?

Reply:
Those strings are used only here. I think that it will be over-engineered if we use final variables or enums here (+ the fact that the JVM will put those literal strings in its string pool).